deploy.yml

```diff
@@ -1,12 +1,13 @@
 ---
-- name: Configure and deploy pythondev apps
-  hosts: sirbot
+
+- name: Configure servers
```

Comment:
🤔 I see your point, none of the base server setup is technically required for the site or sirbot. It just needs python, nginx, ssl, etc. You're right, in our `deploy.yml` it makes sense and then the `sirbot.yml` and `website.yml` playbooks are really just the app setup